backend/infrastructure/api/endpoints/client.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import APIRouter, HTTPException, Request
+from pydantic import BaseModel
+from infrastructure.mongo.client_handler import register_client, get_client
+
+class Client(BaseModel):
+    email: str
+    payment_address: str
+    delivery_address: str
+    nip: str
+    orders: str
+    password: str
+    company_name: str
+
+router = APIRouter()
+
+@router.post("/register")
+async def upload_client_endpoint(client_data: Client, request: Request):
+    if request.headers.get("Content-Type") != "application/json":
+        raise HTTPException(status_code=400, detail="Content-Type must be application/json")
+    
+    try:
+        client_id = register_client(
+            client_data.email,
+            client_data.payment_address,
+            client_data.delivery_address,
+            client_data.nip,
+            client_data.orders,
+            client_data.password,
+            client_data.company_name
+        )
+
+        return {
+            "info": f"Client '{client_data.email}' registered successfully",
+            "client_id": client_id,
+            "email": client_data.email,
+            "payment_address": client_data.payment_address,
+            "delivery_address": client_data.delivery_address,
+            "nip": client_data.nip,
+            "orders": client_data.orders,
+            "password": client_data.password,
```

Comment:
why do we return password to account here? you can easy check it and resuse to login as sb else

Reply:
Exclude sensitive fields payment_address, orders? (we should nt have at start etc.)